src/amo/pages/Guides/index.js

```diff
@@ -251,7 +254,16 @@ export const getSections = ({
   }
 };
 
-export const getContent = (slug: string, i18n: I18nType): GuideType | null => {
+export const getContent = ({
+  slug,
+  i18n,
+  sections,
+}: {|
+  slug: string,
+  i18n: I18nType,
+  sections?: Array<SectionsType>,
+|} = {}): GuideType | null => {
+  const _sections = sections;
```

Comment:
This is a bit confusing because we don't do this usually (i.e. renaming a prop). Why do you need this? Can't you use `_sections` directly so that we know it is mainly used for testing purpose?